.github/workflows/extension-upload.yml

```diff
@@ -40,7 +41,7 @@ jobs:
 
       # Outputs beta.zip and prod.zip
       - name: Zip extension versions
-        run: pnpm dlx tsx apps/extension/src/utils/zip-extension.ts
+        run: pnpm tsx apps/extension/src/utils/zip-extension.ts
```

Comment:
`dlx` is no longer necessary as `tsx` is included in the root package dependencies 

https://github.com/prax-wallet/web/blob/6104222ab39305b4a24548789413b9ec1ba71a85/package.json#L34